src/commands/sarif/upload.ts

```diff
@@ -42,18 +42,18 @@ export class UploadSarifReportCommand extends Command {
       See README for details.
     `,
     examples: [
-      ['Upload all SARIF report files in current directory', 'datadog-ci sarif upload --service my-service .'],
+      ['Upload all SARIF report files in current directory', 'datadog-ci sarif upload.'],
```

Comment:
The 2nd string of each tuple is an example of command line. The `.` isn't punctuation, but refers to the current directory.
```suggestion
      ['Upload all SARIF report files in current directory', 'datadog-ci sarif upload .'],
```